far/editor.cpp

```diff
@@ -125,35 +126,48 @@ eol Editor::GetDefaultEOL()
 	return Global->Opt->EdOpt.NewFileUnixEOL? eol::unix : eol::win;
 }
 
+// $ 2023-01-15 MZK: Move it to global?
```

Comment:
It will introduce new dependency, global on strmix (something) and makes global aware of codepage (though in a trivial way). I was not sure it would be a good idea. Otherwise, these functions should be in global.

Reply:
Moved. If you think the dependency should not be there, let me know, I'll move them back to Editor.